### ISSUE TITLE
Cypress hangs due to xvfb permissions. More intuitive output needed.

### ISSUE THREAD

**Issue by shauvik** [cypress-io/cypress#330]
**Problem:**
On a mac, `./node_modules/cypress-cli/bin/cypress -v` was hanging for a coworker.

By tracing, we found that xvfb that cypress uses internally didn't have permissions for creating `/tmp/.X11-unix `
 ```
~/src/mn/projects/fullstory xvfb
_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
_XSERVTransSocketUNIXCreateListener: mkdir(/tmp/.X11-unix) failed, errno = 2
_XSERVTransMakeAllCOTSServerListeners: failed to create listener for local
(EE) 
Fatal server error:
(EE) Cannot establish any listening sockets - Make sure an X server isn't already running(EE) 
```

**Manual Solution:** 
Run `sudo xvfb` one time, it will create `/tmp/.X11-unix`, and after that `xvfb` can run
And then `./node_modules/cypress-cli/bin/cypress --version` doesn’t hang anymore

**Comment by jennifer-shehane** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
